$NetBSD: patch-D8487,v 1.1 2020/05/24 01:16:26 joerg Exp $

# HG changeset patch
# User Joerg Sonnenberger <joerg@bec.de>
# Date 1587944362 -7200
#      Mon Apr 27 01:39:22 2020 +0200
# Branch stable
# Node ID 5d309906ed0e4e2b0ade4d1539cbea7dc2c44771
# Parent  a50f33f1ff24466d6d721685cc65a732cb4c522c
fastexport: downgrade message about already exported changesets to debug

The old warning level is just too noisy for incremental processing.

Differential Revision: https://phab.mercurial-scm.org/D8487

diff -r a50f33f1ff24 -r 5d309906ed0e hgext/fastexport.py
--- hgext/fastexport.py	Sun Apr 26 14:29:47 2020 -0400
+++ hgext/fastexport.py	Mon Apr 27 01:39:22 2020 +0200
@@ -80,7 +80,7 @@
     ctx = repo[rev]
     revid = ctx.hex()
     if revid in marks:
-        ui.warn(_(b"warning: revision %s already exported, skipped\n") % revid)
+        ui.debug(b"warning: revision %s already exported, skipped\n" % revid)
         return
     parents = [p for p in ctx.parents() if p.rev() != nullrev]
     for p in parents:
